src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/NotificationPopUps.kt

```diff
@@ -10,7 +10,7 @@ import java.time.Duration
 
 
 internal class NotificationPopUps(private val driver: WebDriver) {
-    private val auiFlagCloseLocator = By.cssSelector(".aui-flag .icon-close")
+    private val auiFlagCloseLocator = By.cssSelector(".aui-flag .icon-close, .aui-flag .aui-close-button")
```

Comment:
Does it work for pre-AUI 9 Jira versions?

Reply:
Yup. We have ITs on 7.x and 8.0. Plus, I verified it manually on `8.12-SNAPSHOT` (with AUI 9) and `8.5.x (pre AUI 9).

Reply:
In CSS, the comma gives an alternative selector. So anything that matched before will still match. This only adds support to AUI 9 without removing the old one.